### DISCUSSION TITLE
Add webhook custom fields

### DISCUSSION THREAD

**Discussion by Fury76** [grafana/grafana#53994] [Alerting]
Hi,
Can you add a new feature to the webhook that can support custom fields and values, so that you can use the webhook to meet many scenarios. For example, if there is a new im system, you only need to support the access of the im system according to the corresponding fields of the im system and the template, without the need for separate development

like this:
![image](https://user-images.githubusercontent.com/7465355/185774341-c85115c5-fa45-4c89-8744-a264bda251fb.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
